platforms/springboot/src/main/java/io/hawt/springboot/HawtioPlugin.java

```diff
@@ -26,12 +27,31 @@ public class HawtioPlugin {
     private Boolean bustRemoteEntryCache;
     private String pluginEntry;
 
+    /**
+     *
+     * @param url The URL of the remote entry for the plugin. This must be the same location as the Hawtio console.
+     * @param scope The name of the container defined at Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     * @param module The path exposed from Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     */
     public HawtioPlugin(String url, String scope, String module) {
         this.url = url;
         this.scope = scope;
         this.module = module;
     }
 
+    /**
+     *
+     * @param scope The name of the container defined at Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     * @param module The path exposed from Webpack ModuleFederationPlugin. See also: sample-plugin/craco.config.js
+     *
+     * This constructor automatically uses hawtio address.
+     */
+    public HawtioPlugin(String scope, String module){
```

Comment:
I don't think it's a good API design that the two constructors have different meanings for the arguments with the same positions and type.
```
HawtioPlugin(String url, String scope, String module)
HawtioPlugin(String scope, String module)
```
If we want to provide such two constructors, I think it's better to move the position of `url` to the end for the first one. When doing so, make sure to update all the places where the original constructor is used.

Also, the 2nd one should be implemented using the 1st (most versatile) one like this:
```
    this(scope, module, "");
```